modules/oe_content_publication/config/install/field.field.node.oe_publication.body.yml

```diff
@@ -0,0 +1,22 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.body
+    - node.type.oe_publication
+  module:
+    - text
+id: node.oe_publication.body
+field_name: body
+entity_type: node
+bundle: oe_publication
+label: Body text
+description: ''
+required: false
+translatable: false
+default_value: {  }
+default_value_callback: ''
+settings:
+  display_summary: 0
+  required_summary: false
+field_type: text_long
```

Comment:
```suggestion
settings:
  display_summary: false
  required_summary: false
field_type: text_with_summary
```

Reply:
This is described as `Type: long text (without summary)` in the task. Why should we have to change the field_type?

Reply:
Alexandra is right, `field_type: text_with_summary` is used for all body fields.
`display_summary: 0` is wrong because it has to be boolean, not integer.

Reply:
This is fixed.